sdk/computeschedule/Azure.ResourceManager.ComputeSchedule/README.md

```diff
@@ -0,0 +1,80 @@
+# Microsoft Azure ComputeSchedule management client library for .NET
+
+**[Describe the service briefly first.]**
```

Comment:
Update the description of your service here.

Reply:
Updated